react/tic-tac-toe/src/app/screens/App/components/Login/components/Field/styles.module.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../../../../../assets/variables';
+@import '../../../../../../../assets/colors';
+
+.fieldContainer {
+  margin-top: 1.5rem;
```

Comment:
It's nice to use `rem` but try to use `px` since: 
1. It's the Wolox standard. 
2. It makes it easier to comply with any design guideline.